### DISCUSSION TITLE
client- broken (repaired itself probably some weird internet erros coused by windows autouppdate)

### DISCUSSION THREAD

**Discussion by micpir99** [fedorovvl/tso_client#175] [General]
after 09.02 loading bar stopped in half progress. Even after 10 min won't move. ticking off/on swf is not helping.
downloading older wersion don't work and removing temp folder and redownloading it don't work too. Can i ask for help?
i hope attached screen will help 
![image](https://user-images.githubusercontent.com/125140405/218266001-189e3954-e56c-41e8-a85d-6a8e09a0d152.png)

**Comment by micpir99** [comment]
2-nd cliend works but i like this one.

**Comment by micpir99** [comment]
it is working again i don't know what helped, but i did a lot of things.

**Comment by aleks7008** [comment]
2.03 had the same problem. the client is not loaded.
is there a solution?

**Reply by fedorovvl** [reply]
unity loads ok? it using same files

**Comment by aleks7008** [comment]
unity loads ok 
(Win 11)

**Reply by fedorovvl** [reply]
maybe smth with internet connection.. don't know :(
[micpir99](https://github.com/micpir99) fixed it somehow

**Reply by micpir99** [reply]
Tbh i didn't fix it it sometimes happens to me

**Reply by micpir99** [reply]
But i think it is somethimg about uppdates of client + hotspoting internet with phone

**Reply by micpir99** [reply]
If you use hotspot first turn off internet on phone, reboot phone and reset computer turn on inernet (on phone) few times thet turn on hotspot.(i think it was something with internet on phone after reset).
I am not 100% shure if it will work
